### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by annanay25**
Signed-off-by: Annanay <annanayagarwal@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,6 +77,42 @@ import (
...
+```
+make lint
+```
+before submitting your PR to catch any linting errors. Linting can be fixed using
+
+```
+make fmt
+```
+
+However, do note that the above command requires the `gofmt` and `goimports` binaries accessible via `PATH`.
```
Comment (mapno) [inline CONTRIBUTING.md:114]:
```suggestion
However, do note that the above command requires the `gofmt` and `goimports` binaries accessible via `$PATH`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,6 +77,42 @@ import (
...
+A CI job runs these tests on every PR.
+
+### Linting
+
+Make sure to run
+
+```
+make lint
+```
+before submitting your PR to catch any linting errors. Linting can be fixed using
```
Comment (mapno) [inline CONTRIBUTING.md:108]:
```suggestion

before submitting your PR to catch any linting errors. Linting can be fixed using
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -77,6 +77,42 @@ import (
...
+### Instrumentation
+
+- **Metrics**: Tempo is instrumented with [Prometheus metrics](https://prometheus.io/). It emits RED metrics for most
+  services and backends. The relevant dashboards can be found in the [Tempo mixin](operations/tempo-mixin).
+- **Logs**: Tempo uses the [go-kit level logging library](https://pkg.go.dev/github.com/go-kit/kit/log/level) and emits
+  logs in the `key=value` (logfmt) format.
+- **Traces**: Tempo uses the [Jaeger Golang SDK](https://github.com/jaegertracing/jaeger-client-go) for tracing instrumentation.
+  As of this writing, only the read path of tempo is instrumented for tracing.
+
+### Testing
```
Comment (mapno) [inline CONTRIBUTING.md:89]: Should we mention also `docker-compose`, `helm` and `tk` as testing too?

Comment (annanay25) [reply CONTRIBUTING.md:89]: Yeah that's a good point. I'll add something in here.

**Review by mapno — APPROVED** [review]
Nice! LGTM

**Pull request merged**